app/javascript/data/forest-types.json

```diff
@@ -3,24 +3,56 @@
     "label": "Plantations",
     "value": "plantations",
     "metaKey": "gfw_plantations",
-    "global": true
+    "global": true,
+    "datasets": [
+      {
+        "dataset": "bb1dced4-3ae8-4908-9f36-6514ae69713f",
+        "layers": ["b8fb6cc8-6893-4ae0-8499-1ca9f1ababf4"]
+      }
+    ]
   },
   {
     "label": "Intact Forest Landscapes",
     "value": "ifl",
     "metaKey": "intact_forest_landscapes_change",
-    "global": true
+    "global": true,
+    "datasets": [
+      {
+        "dataset": "13e28550-3fc9-45ec-bb00-5a48a82b77e1",
+        "layers": ["fd44b976-62e6-4072-8218-8abf6e254ed8"]
+      }
+    ]
   },
   {
     "label": "Primary Forests",
     "value": "primary_forest",
     "metaKey": "primary_forest",
-    "global": true
+    "global": true,
+    "datasets": [
+      {
+        "dataset": "714339c1-c775-4303-aad4-16d975b2f023",
+        "layers": ["079fae08-5696-4926-9417-794bd3a7e8dc"]
+      }
+    ]
   },
   {
     "label": "Mangrove forests",
     "value": "mangroves",
     "metaKey": "mangrove_2010_gmw",
-    "global": true
+    "global": true,
+    "datasets": [
+      {
+        "dataset": "b3b8ca9d-a071-4383-b898-d7f64573b51f",
+        "layers": [
+          "49e3f955-2bab-4f73-8589-c9c039b8da29",
```

Comment:
this is a tricky one. Each mangrove layer here is for a specific year and by category too. We should only show one layer unless we give them control over which layer is displaying. And ideally the layer that is used to intersect with the data. As the key above specifies that 2010 is the year they are using, let only put the 2010 layer.